docs/conf.py

```diff
@@ -96,8 +97,14 @@
     # "navbar_align": "left",  # [left, content, right] For testing that the navbar items align properly
     # "navbar_start": ["navbar-logo", "navbar-version"],
     # "navbar_center": ["navbar-nav", "navbar-version"],  # Just for testing
-    # "navbar_end": ["navbar-icon-links", "navbar-version"]  # Just for testing
+    "navbar_end": ["version-switcher", "navbar-icon-links"],
     # "footer_items": ["copyright", "sphinx-version", ""]
+    "switcher": {
+        "json_url": "/_static/switcher.json",
+        # "json_url": "https://pydata-sphinx-theme.readthedocs.io/en/latest/_static/switcher.json",
```

Comment:
This is the "last change before merge" suggestion (because it will break the switcher functionality on the CI PR build)
```suggestion
        "json_url": "https://pydata-sphinx-theme.readthedocs.io/en/latest/_static/switcher.json",
```

Reply:
bump?  I think I've adressed all feedback here, and all that remains is for someone with merge rights to sign off on [the new section in the rendered docs](https://pydata-sphinx-theme--436.org.readthedocs.build/en/436/user_guide/configuring.html#add-a-dropdown-to-switch-between-docs-versions), and if they're happy, apply this suggestion and then merge.

Reply:
reminder to apply this suggestion before merging; it will break the switcher in the PR doc build but is needed for the switcher to work on the real docs.

Reply:
Not sure if I understood it right or if I am missing anything in this lengthy context. Can someone please help me with the latest repo with both versions and language drop-down implementation for the RTD theme in Sphinx? @drammock , @astrojuanlu, or @humitos

Reply:
@Sachin-Suresh I'm not sure to understand what is your question. I'd recommend you to open a new issue with a longer and detailed description. Otherwise, it'd be hard to parse your question in a PR that has ~150 comments already and 1 year old.